tests/utils.go

```diff
@@ -776,3 +779,15 @@ func RunVMAndExpectLaunchWithRunStrategy(virtClient kubecli.KubevirtClient, vm *
 
 	return updatedVM
 }
+
+func CreateBlockDataVolume(virtClient kubecli.KubevirtClient) (*cdiv1.DataVolume, error) {
+	sc, foundSC := libstorage.GetBlockStorageClass(k8sv1.ReadWriteOnce)
+	Expect(foundSC).To(BeTrue(), "Block storage is not present")
```

Comment:
Why do you expect the Block storage to be always present? What will happen on the HPP / NFS / FSx / Portworx lane?

Reply:
It will fail if it is not present, but the label for [req-storage] should make it run on block storage clusters, In today condition, the tests were skipped if it was not present which in turn would lead to us missing failures, do you think there is a better way to not skip those tests?